### DISCUSSION TITLE
Add-PnPTeamsTab of type SharePointPageAndList for picture library not working

### DISCUSSION THREAD

**Discussion by achimismaili** [pnp/powershell#404] [Q&A] [answered]
I have a problem with a picture library, in that I cannot add it to Teams as a SharePointPageAndList, because it complains, that the parameter -ContentUrl is not known, instead I use now Type WebSite:

_This does not work with picture library_
`Add-PnPTeamsTab -Team $teamName -Channel $channel.DisplayName -DisplayName $fotoLibName -Type SharePointPageAndList -ContentUrl "https://$Tenant.sharepoint.com$($view.ServerRelativeUrl)"`

_This works fine with picture library_
`Add-PnPTeamsTab -Team $teamName -Channel $channel.DisplayName -DisplayName $fotoLibName -Type WebSite -ContentUrl "https://$Tenant.sharepoint.com$($view.ServerRelativeUrl)"`

Is a picture library not supported for type SharePointPageAndList in MS Teams?
And if it is not supported, why does running the command complain, that a parameter is not known!?

- At least through the Teams UI, I **can** add a PictureLibrary through the add tab option/app "SharePoint" with the option "Any SharePoint site" ...

### What is the version of the Cmdlet module you are running?
1.3.0

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by veronicageek** [comment, marked as answer]
Hi @achimismaili - Please have a look at the answer provided by Erwin for Issue #225, it should clarify why.
